cmd/kubeadm/app/features/features_test.go

```diff
@@ -207,6 +208,11 @@ func TestCheckDeprecatedFlags(t *testing.T) {
 			features:    map[string]bool{"feature1": true},
 			expectedMsg: map[string]string{},
 		},
+		{
+			name:        "invalid feature",
+			features:    map[string]bool{"": true},
```

Comment:
empty FG is a special one, and how about add a general invalid FG like 'feature2' or 'invalid'.